### PR TITLE
[TEST] intercept adds flag to avoid segment fault

### PR DESCRIPTION
**Pull request by Tricky511**

### CODE REVIEW

File: test_io500.sh [hunk 1]
```diff
@@ -12,6 +10,30 @@ function green_font() {
...
+    result=$[ $end_time - $start_time ]
+    echo -e "fuse all tests ok, cost: $(green_font ${result}ms)"
+    sudo umount ~/fs
+    sudo rm -rf ~/fs
+}
+
+function intercept_test() {
+    start_time=$[$(date +%s%N)/1000000]
+    cd io500
+    SEALFS_CONFIG_PATH=../examples SEALFS_MOUNT_POINT=/home/luan/fs LD_PRELOAD=../target/debug/libintercept.so timeout -s SIGKILL 200 mpirun -np 2 ./io500 config-minimal.ini
```
Comment (luanshaotong) [inline test_io500.sh:30]: use ~/fs

File: test_io500.sh [hunk 2]
```diff
@@ -53,11 +72,8 @@ echo "" >> config-minimal.ini
...
-timeout -s SIGKILL 200 mpirun -np 2 ./io500 config-minimal.ini
-
 cd ..
+fuse_test
+intercept_test
 
-end_time=$[$(date +%s%N)/1000000]
-result=$[ $end_time - $start_time ]
-echo -e "all tests ok, cost: $(green_font ${result}ms)"
 finish 0
```
[the remark this answers is not in the document]
Comment (luanshaotong) [reply test_io500.sh:79]: like
```
set +e

fuse_test
result1=$?

intercept_test
result2=$?

set -e

result=$((result1||result2))
finish $result
```

[the remark this answers is not in the document]
Comment (Tricky511) [reply test_io500.sh]: done

File: test_io500.sh (hunk 1 above)
Comment (Tricky511) [reply test_io500.sh]: done

**Pull request merged**